packages/salesforcedx-vscode-lwc/resources/static/js-meta.xsd

```diff
@@ -288,11 +292,18 @@
           <xs:annotation>
             <xs:documentation>Use a separate targetConfig for each different page type configuration.</xs:documentation>
           </xs:annotation>
-          <xs:sequence>
-            <xs:element type="objects" name="objects" minOccurs="0" />
-            <xs:element type="property" name="property" minOccurs="0" maxOccurs="unbounded" />
-            <xs:element type="supportedFormFactors" name="supportedFormFactors" minOccurs="0" />
-          </xs:sequence>
+          <xs:choice minOccurs="0" maxOccurs="unbounded">
+            <xs:sequence>
+              <xs:element type="xs:boolean" name="hasStep" minOccurs="0">
+                <xs:annotation>
+                  <xs:documentation>Specify that your component requires an attached step to function as expected. Only valid for `analytics__Dashboard` targets. With this set to `true`, the Tableau CRM dashboard builder UI prompts you to attach an existing step or create a new step when creating an instance of your component. Components with an attached step have access to step-specific properties like `results` and `selection`.</xs:documentation>
```

Comment:
vscode-xml extension handles markdown in `xs:documentation` in the hovers, and markdown works fine as plain text if any other tool ever looks at this xsd.